python/icub_models/icub_models.py

```diff
@@ -0,0 +1,29 @@
+# @file icub_models.py
+# @authors Giulio Romualdi
+# @copyright 2022 Istituto Italiano di Tecnologia Released under the terms of the Creative Commons Attribution Share Alike 4.0 International
+
+from os import listdir
+from os.path import dirname, join, isdir, exists, realpath
+from typing import List
+
+
+def get_models_path() -> str:
+    relative_path = '@ICUB_MODELS_CXX_PYTHON_INSTALL_RELATIVE_DIR@'
+    # If relative_path starts with @, it is a not escaped string
+    # from CMake, i.e. that means that the file was installed via setup.py
+    # let's substitute relative_path with its default value
+    if relative_path.startswith('@'):
+        relative_path = '../../../..'
```

Comment:
Is it working also on Windows? I'm referring to the usage of `/`

Reply:
Yes, see https://github.com/traversaro/icub-models/commit/011a877a511bba66892c84a1c55c040c91adfe0d#r69749781 .

Reply:
Actually this is does not work on windows when installing with `pip install .` . Not that it is a workflow I am super-worried of, but it may be an indication we are doing something wrong. Not sure if @diegoferigo can lend a quick eye on the setup.py/Python part (I already checked the CMake part and I am quite sure on that part).

Reply:
I didn't check in great detail. In order to get better multiplatform support especially for  `/` vs `\`, I warmly recommend using `pathlib`. It allows to use the `/` _operator_ to separate paths, that is translated automatically to a real path depending on the OS.

Reply:
> In order to get better multiplatform support especially for `/` vs `\`, I warmly recommend using [`pathlib`](https://docs.python.org/3/library/pathlib.html).

Just to understand, why is this necessary? `/` works on both Windows and Linux/macOS".

Reply:
> > In order to get better multiplatform support especially for `/` vs `\`, I warmly recommend using [`pathlib`](https://docs.python.org/3/library/pathlib.html).
> 
> Just to understand, why is this necessary? `/` works on both Windows and Linux/macOS.

AFAIK this code was working on linux but not on windows. Just a wild guess considering the few lines of code. Disclaimer: didn't test the code myself.

Reply:
I tested the code on Windows and `/` works fine on Windows (even with mixed with `\`).

Reply:
The test is failing finding:

```
C:/hostedtoolcache/windows/Python/3.8.10/share/iCub/robots
```

It seems that a `icub_models` is missing, I'd expect:

```
C:/hostedtoolcache/windows/Python/3.8.10/*icub_models*/share/iCub/robots
```